internal/business/persistedoperations/dir_loader.go

```diff
@@ -49,6 +64,8 @@ func (d *DirLoader) Load(_ context.Context) (map[string]PersistedOperation, erro
 				continue
 			}
 
+			fileLoaderCounter.WithLabelValues().Inc()
```

Comment:
This will only ever increment, do you not want a gauge and do a `Set` after the for loop, keep a counter to count the # of files processed?

Reply:
let me change that